framework/UPGRADE.md

```diff
@@ -51,6 +51,12 @@ if you want to upgrade from version A to version C and there is
 version B between A and C, you need to follow the instructions
 for both A and B.
 
+Upgrade from Yii 2.0.42
+-----------------------
+
+* `yii\base\ErrorHandler` does not expose the `$_SERVER` information implicitely anymore. If your application still expects
+  the presence of the information you can set `YII_DEBUG` to true.
```

Comment:
That's not a good advice in case the application expects it in production mode. I'd just remove this "If".